src/components/appMenuSections/profile/MinerProfile.tsx

```diff
@@ -195,7 +199,7 @@ const MinerProfile = () => {
             type="number"
             onChange={(e) => {
               const inputAmount = e.target.value;
-              const inputAmountToInt = parseInt(inputAmount);
+              const inputAmountToInt = parseFloat(inputAmount);
```

Comment:
why is it named "ToInt". it refers to the amount of stx.

rename it. by doing it here, would appear conflicts in the following merges, should we only rename in the last commit? @Alexandrescu

Reply:
https://github.com/stacks-degens/starters-front-end/issues/56